.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+name: Create version packages PR or release
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch: # Allow manual triggering of the workflow
+    inputs:
+      tag:
+        description: "Tag to release"
+        required: true
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      id-token: write
+    steps:
+      - name: Set tag name
+        id: set-tag
+        run: echo "TAG_NAME=${{ github.event.release.tag_name || github.event.inputs.tag }}" >> $GITHUB_ENV
+
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.SHOPIFY_GH_ACCESS_TOKEN }}
+          # Use the tag associated with the release
+          ref: ${{ env.TAG_NAME }}
+
+      - name: Setup Node from .nvmrc
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
```

Comment:
You also need to add the following config here:
```yaml
registry-url: 'https://registry.npmjs.org'
```
Otherwise the NPM credentials won't be correctly applied.